src/compiler/emitter.ts

```diff
@@ -4766,9 +4766,14 @@ const _super = (function (geti, seti) {
                 emitToken(SyntaxKind.CloseBraceToken, body.statements.end);
             }
 
-            function findInitialSuperCall(ctor: ConstructorDeclaration): ExpressionStatement {
+            /**
+             * Return the statement at a given index if it is a super-call statement
+             * @param ctor constructor declaration
+             * @param index an index to constructor's body to check
+             */
+            function getSuperCallAtGivenIndex(ctor: ConstructorDeclaration, index: number): ExpressionStatement {
                 if (ctor.body) {
-                    const statement = (<Block>ctor.body).statements[0];
+                    const statement = (<Block>ctor.body).statements[index];
                     if (statement && statement.kind === SyntaxKind.ExpressionStatement) {
                         const expr = (<ExpressionStatement>statement).expression;
                         if (expr && expr.kind === SyntaxKind.CallExpression) {

```

Comment:
Actually, `expression` isn't optional, you should be able to write `isSuperCallExpression(expr)`.